pi-geofence/src/androidTest/java/com/ibm/pisdk/geofencing/test/TestUtils.java

```diff
@@ -0,0 +1,71 @@
+/**
+ * Copyright (c) 2015-2016 IBM Corporation. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+package com.ibm.pisdk.geofencing.test;
+
+import android.content.Context;
+import android.support.test.InstrumentationRegistry;
+
+import java.util.Random;
+
+/**
+ *
+ */
+public class TestUtils {
+  /**
+   * A global PRNG.
+   */
+  private static final Random RAND = new Random(System.nanoTime());
```

Comment:
might want to remove some of this information; TENANT_ID, APP_ID, API_KEY. And provide how to retrieve that information for their own instance to test.